### DISCUSSION TITLE
請問可否增加預設所有網站bypass, 只有特定網站(加入白名單)才執行的模式

### DISCUSSION THREAD

**Discussion by bigfrog123** [F9y4ng/GreasyFork-Scripts#393] [问答专区(Question / Answer)]
目前是預設所有網站都運行, 特定網站透過排除(加入黑名單)不執行, 小弟的需求剛好相反, 目前版本有辦法改以白名單為主嗎? 謝謝!

**Comment by F9y4ng** [comment]
Duplicate of #334

**Reply by bigfrog123** [reply]
這解法雖然可以, 但還是不太一樣。
因為這樣會變成所有要運行的網站都要設置並儲存一份自訂的 "字體渲染設置", 而不是只要將網站"加入渲染"(使用全局的渲染設置)
若能將 "預設是否運行" 移出全局設定(讓全局設定只包含預設的字體渲染設置)
譬如: 網站過濾模式(運行模式): 白名單(預設不執行渲染)或黑名單(預設執行渲染)
這樣切換模式時, 也不須清除設定重來, 網站自訂設置也不影響。提供參考。

**Reply by F9y4ng** [reply]
建议非常好，但你忽略了一件事，我很懒。
